indexing-service/src/test/java/org/apache/druid/indexing/seekablestream/SeekableStreamSupervisorSpecTest.java

```diff
@@ -750,16 +757,23 @@ public void testSeekableStreamSupervisorSpecWithScaleOut() throws InterruptedExc
     EasyMock.expect(taskMaster.getSupervisorManager()).andReturn(Optional.absent()).anyTimes();
     EasyMock.replay(taskMaster);
 
-    TestSeekableStreamSupervisor supervisor = new TestSeekableStreamSupervisor(3);
+    Capture<ServiceMetricEvent.Builder> metricCapture = Capture.newInstance(CaptureType.ALL);
+    ServiceEmitter dynamicActionEmitter = EasyMock.mock(ServiceEmitter.class);
```

Comment:
No need to mock this, just use a `StubServiceEmitter`. It has utility methods to help verify the emitted values.